stf/tasks/optimize_hyperparameters.py

```diff
@@ -23,7 +23,7 @@
 
 def main():
     with TaskContext(__file__) as context:
-        model_type = ["xgb", "xgb_quantile"]
```

Comment:
xgb shoud remain xgb but lgb should be added to the list

Reply:
I'll change it, thanks!